app/code/Magento/Checkout/Test/Mftf/Test/StorefrontPersistentDataForGuestCustomerWithPhysicalQuoteTest.xml

```diff
@@ -49,8 +49,7 @@
         <waitForLoadingMaskToDisappear stepKey="waitForLoadingMaskToDisappearAfterFlatRateSelection"/>
         <see selector="{{CheckoutCartSummarySection.total}}" userInput="15" stepKey="assertOrderTotalField"/>
         <!-- 5. Refresh browser page (F5) -->
-        <reloadPage stepKey="reloadPage"/>
-        <waitForPageLoad stepKey="waitForPageLoad"/>
```

Comment:
We still may consider this change as BiC (there might be extensions with before/after references to this step key). The common (a little bit dirty but still common) practice, in this case, will be replacing the old action with a comment that has the same `stepKey`. Example is below 

```xml
<comment userInput="Adding comment to replace reload action and preserve Backward Compatibility" stepKey="waitForPageLoad" />
```

Hope it makes sense.

Reply:
Hi @rogyar 
**Done**
Thank you for advice.